src/widget/wmainmenubar.cpp

```diff
@@ -168,6 +168,34 @@ void WMainMenuBar::initialize() {
 
     pLibraryMenu->addSeparator();
 
+    QString searchHereTitle = tr("Search in Current View...");
+    QString searchHereText = tr("Search for tracks in the current library view");
+    auto* pSearchHere = new QAction(searchHereTitle, this);
+    pSearchHere->setShortcut(QKeySequence(m_pKbdConfig->getValue(
+            ConfigKey("[KeyboardShortcuts]", "LibraryMenu_SearchInCurrentView"),
+            tr("Ctrl+f"))));
+    pSearchHere->setShortcutContext(Qt::ApplicationShortcut);
+    pSearchHere->setStatusTip(searchHereText);
+    pSearchHere->setWhatsThis(buildWhatsThis(searchHereTitle, searchHereText));
+    connect(pSearchHere, &QAction::triggered, this, &WMainMenuBar::searchInCurrentView);
+    pLibraryMenu->addAction(pSearchHere);
+
+    QString searchAllTitle = tr("Search in Tracks Library...");
```

Comment:
Let's use "Tracks view" so it's consistent with "Current view" and the searchbar tooltip